index.html

```diff
@@ -115,39 +136,41 @@ <h1 class="h1about">About</h1>
             <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
             <button class="cvBut" type="button">Get my CV <img src="./images/download.png" alt="Download CV"><img src="./images/download-blue.png" alt="Download CV"></button>
         </div>
-        <article class="languages">
-            <h3>Languages</h3>
-            <ul class="lang-list">
-                <li class= "about-list">JavaScript(ES5/ES6)</li>
-                <li class= "about-list">Ruby</li>
-                <li class= "about-list">HTML</li>
-                <li class= "about-list">CSS</li>
-            </ul>
-            <img class="hash" src="./images/hash.png" alt="">
-        </article>
+        <div class="colorboxs">
+            <article class="languages">
+                <h3>Languages</h3>
+                <ul class="lang-list">
+                    <li class= "about-list">JavaScript(ES5/ES6)</li>
+                    <li class= "about-list">Ruby</li>
+                    <li class= "about-list">HTML</li>
```

Comment:
- [ ] Kindly implement the suggestion made by the other reviewer for this section. If you look at your design, you will see your coloured boxes are vertical but that of the original design is two on a row and two on a column. The language's coloured box is stacked net to the framework's coloured box while that of the skill's box is under that of the language's box. kindly implement it that way. 👍